src/app/settings/print-settings/print-settings.component.ts

```diff
@@ -0,0 +1,189 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { PrintOptions } from '../../shared/models/printing';
+import { Settings } from '../../shared/models/settings';
+import { PrintOptionsMenuService } from '../../shared/print-options-menu/print-options-menu.service';
+
+@Component({
+  selector: 'app-print-settings',
+  templateUrl: './print-settings.component.html',
+  styleUrls: ['./print-settings.component.css']
+})
+export class PrintSettingsComponent implements OnInit {
+  @Input()
+  settings: Settings;
+  @Output('savePrintSettings')
+  savePrintSettings = new EventEmitter<boolean>();
+
+  selectAllOptions: boolean = true;
+
+  printOptions: PrintOptions;
+  printOptionsSub: Subscription;
+
+  constructor(private printOptionsMenuService: PrintOptionsMenuService) { }
```

Comment:
Don't need to inject PrintOptionsMenuService. This is initializing it at a global/shared scope and so it won't properly initialize and get global settings when print-options-menu.comp opens in fsat/psat etc..